app/views.py

```diff
@@ -42,6 +42,27 @@ def getUser():
     return user
 
 
+def getProjects(username):
+    if not isLoggedIn():
+        return [{'error': "Not logged in."}]
+
+    ts = pymesync.TimeSync(baseurl=app.config['TIMESYNC_URL'],
+                           test=app.config['TESTING'],
+                           token=session['token'])
+    projects = ts.get_projects()
```

Comment:
I think this should only return projects that the current user has permissions on, in which case we can skip checking for the user in each project... @LyonesGamer can you confirm?